Common/UnitDefinitions/Volume.json

```diff
@@ -245,6 +245,21 @@
         }
       ]
     },
+    {
+      "SingularName": "CubicAngstrom",
+      "PluralName": "CubicAngstroms",
+      "BaseUnits": {
+        "L": "Angstrom"
+      },
+      "FromUnitToBaseFunc": "{x} * 1E-30",
+      "FromBaseToUnitFunc": "{x} / 1E-30",
+      "Localization": [
```

Comment:
Looks good, but you are missing implementing the test cases for the new units, so the build currently fails.

See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit